modules/images-cluster-sample-imagestream-import.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+// * openshift_images/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="images-cluster-sample-imagestream-import_{context}"]
+= Configuring periodic importing of Cluster Sample Operator image stream tags
+
```

Comment:
Should we put an introductory paragraph to tell folks what they are doing and why?

Reply:
That would be good to have.